b_cfn_elasticsearch_index/function.py

```diff
@@ -0,0 +1,69 @@
+from __future__ import annotations
```

Comment:
We dont support Python2+ code, hence this is not necessary.

Reply:
This is for delayed annotation processing, not for Python 2 support, it is required for some annotations in this file to work as-is, otherwise they would need to be quoted.